packages/alias/package.json

```diff
@@ -22,9 +22,9 @@
     "ci:lint:commits": "commitlint --from=${CIRCLE_BRANCH} --to=${CIRCLE_SHA1}",
     "ci:test": "pnpm run test -- --verbose",
     "lint": "pnpm run lint:js && pnpm run lint:docs && pnpm run lint:package",
-    "lint:docs": "prettier --single-quote --arrow-parens avoid --trailing-comma none --write README.md",
```

Comment:
Having different rules for readmes and the code made things more complicated, so I made them the same. If we want to keep them different more work is needed

Reply:
See https://prettier.io/docs/en/configuration.html#configuration-overrides